.github/workflows/tests.yml

```diff
@@ -26,18 +26,17 @@ jobs:
         - name: Python 3.12 (Ubuntu), with doctests, skip slow
           os: ubuntu-latest
           python: '3.12'
-          toxenv: py312-pins-doctest
-          toxargs: --develop
```

Comment:
I had thought that the `--develop` flag here was to enable Python development mode, but I think this is just passed to tox.